src/main/resources/messages_en_US.properties

```diff
@@ -480,6 +480,30 @@ home.removeImagePdf.title=Remove image
 home.removeImagePdf.desc=Remove image from PDF to reduce file size
 removeImagePdf.tags=Remove Image,Page operations,Back end,server side
 
+#replace-invert-color
+replace-color.title=Replace-Invert-Color
```

Comment:
Please add to en_GB file as well before merging

Reply:
added as per request